scripts/paths.js

```diff
@@ -1,28 +1,11 @@
-import locales from '../locales/index.js';
-import pages from '../data/pages.js';
-var localeString = ':locale';
+var locales = require('../locales/index.js');
```

Comment:
I'd do it in this patch. And to be clear, I'm talking about the 'scripts/paths.js' file, not the locale data